.github/workflows/ci.yml

```diff
@@ -122,3 +122,8 @@ jobs:
     - name: Build and Run QEMU tests
       working-directory: firmware/qemu
       run: ./test.sh
+    - name: Backward compatibility check against decoder v0.1.0
+      working-directory: firmware/qemu
+      run: |
+        cargo install --debug --git https://github.com/knurling-rs/defmt --branch do-not-delete-v0.1.0-with-no-version-check qemu-run
```

Comment:
We could also use a tag instead of a branch here, to reduce branch clutter and make the installed version immutable. Up to you though, this also seems fine.

Reply:
certainly a tag sounds better! had not thought about that option. will update this